src/Map.ts

```diff
@@ -1424,6 +1474,213 @@ export class Map extends maplibregl.Map {
       options.beforeId,
     );
 
-    return retunedInfo;
+    return returnedInfo;
+  }
+
+  /**
+   * Add a polygon with styling options.
+   */
+  addPolygon(
+    // this Feature collection is expected to contain on LineStrings and MultilLinestrings
```

Comment:
```suggestion
    // this Feature collection is expected to contain on LineStrings and MultiLinestrings
```

lint

I also really don't know what the correct one is here:
* FeatureCollection
* Feature Collection
* Feature collection

Up to you.

Reply:
Yeah i think my `Feature collection` is mix between the type name `FeatureCollection` and the English of it being a `feature collection`. But the truth is, it actaully does not have to be of type "FeatuerCollection"